frontend/packages/ceph-storage-plugin/src/components/dashboards/object-service/data-consumption-card/data-consumption-card-dropdown.tsx

```diff
@@ -142,6 +142,7 @@ export const DataConsumptionDropdown: React.FC<DataConsumptionDropdownProps> = (
           placeholderText={t('ceph-storage-plugin~Type: {{selectedService}}', {
             selectedService,
           })}
+          aria-label={t('ceph-storage-plugin~Break By Dropdown')}
```

Comment:
Not sure if I'm understanding the context here but what does this (`Break by Dropdown`) mean?

Reply:
please check `Capacity breakdown` card...
Similar label is used at other places as well:
https://github.com/openshift/console/blob/10788fddfaaed31f0b5b64d34690d5a5f324652a/frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/breakdown-card.tsx#L85

Reply:
Cool, makes sense.